noxfile.py

```diff
@@ -265,6 +265,7 @@ def local(session):
         *test_dependencies,
         *pyproject_data["project"]["optional-dependencies"]["ssh"],
         *pyproject_data["project"]["optional-dependencies"]["nox"],
+        "-e",
```

Comment:
NB: Noticed during local testing that changes to vectors weren't being propagated between runs here; adding `-e` fixes the problem (and is similarly used in other environments in the noxfile).